### PR TITLE
fix lint errors

### PR DESCRIPTION
**Pull request by Alexandra1003**
Finish fixing eslint errors

### CODE REVIEW

**Comment by PodstDO** [comment]
Great!

**Review by PodstDO — APPROVED** [review]
Awesome!

**Review by ZelikSV — APPROVED** [review]

File: src/tests/constructor.test.js [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+    expect(arr3[0]).toEqual(1);
+    expect(arr3[1]).toEqual(2);
+    expect(arr4.length).toBe(1);
+    expect(arr4[0]).toEqual('1');
+  });
+
+  test('should have own property length', () => {
+    const arr = new MyArray();
+
+    expect(Object.prototype.hasOwnProperty.call(arr, 'length')).toBeTruthy();
```
Comment (PodstDO) [inline src/tests/constructor.test.js:22]: Very clever solution!

File: src/tests/find.test.js [hunk 2]
```diff
@@ -4,14 +4,14 @@ import MyArray from '../index';
 describe('tests for method find', () => {
   test('returns undefined if there is no such element', () => {
     const arr = new MyArray(2, 4, 5);
-    const callback = (x) => x > 10; 
+    const callback = x => x > 10;
 
     expect(arr.find(callback)).toBeUndefined();
   });
 
   test('returns the first element that satisfies the expression', () => {
```
Comment (PodstDO) [inline src/tests/find.test.js:12]: Useful checking!

**Pull request merged**